velox/common/caching/tests/AsyncDataCacheTest.cpp

```diff
@@ -505,6 +505,9 @@ void AsyncDataCacheTest::loadNFiles(
   for (auto i = 0; i < numLoads; i++) {
     semaphore.acquire();
   }
+  while (cache_->ssdCache()->writeInProgress()) {
```

Comment:
Shall we put this right above the test code that check the TTL stats? Thanks!

Reply:
Revises